### PR TITLE
Document multi-language

### PR DESCRIPTION
**Pull request by gabrielhase**
- update reference docs
- add guide

### CODE REVIEW

**Comment by gabrielhase** [comment]
@peyerluk please review

**Comment by gabrielhase** [comment]
@mthpvg fyi

File: walkthroughs/setup_multilanguage.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    articleList: {
+      displayFilters: ['language']
+    }
+  }
+}
+```
+
+Equivalently for pages if you want to have it there too.
+
+## A word on translations
```
Comment (peyerluk) [inline walkthroughs/setup_multilanguage.md]: How about moving this part of the documentation into it's own file so it is clearly separated from the features we offer in our product?

File: walkthroughs/setup_multilanguage.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+This assumes that for each document you store: where a document was translated from, what the corresponding master document is and what the locale of this document is. Again: just an example.
+
+### Metadata
+
+In order to update your custom index with the data above, this data needs to be on each document inside of Livingdocs. We recommend doing a [custom metadata plugin](./metadata/metadata-examples.md) for this. Your metadata plugin has the job of storing all the information that your custom index needs on the metadata of each document.
+
+### Update index
+
+Next you need a way to update this index whenever something happens in Livingdocs. There are basically 3 use cases:
```
Comment (peyerluk) [inline walkthroughs/setup_multilanguage.md]: 'whenever something happens' this is a bit unspecific :)

Do you meant 'whenever a document gets published or unpublished'?

**Pull request merged**